mw-webapp/src/logic/reportsTable/DayReportsTable.tsx

```diff
@@ -0,0 +1,54 @@
+import {useEffect, useState} from "react";
+import {useParams} from "react-router-dom";
+import {Button} from "src/component/button/Button";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+import {QueryParamTypes} from "src/logic/QueryParamTypes";
+import {columns} from "src/logic/reportsTable/columns";
+import {ReportsTable} from "src/logic/reportsTable/ReportsTable";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Render table with dayReports of specific way
+ */
+export const DayReportsTable = () => {
+  const {uuid} = useParams<QueryParamTypes["uuid"]>();
+
+  const [dayReports, setDayReports] = useState<DayReport[]>([]);
+
+  /**
+   * Gets all day reports
+   */
+  const loadDayReports = async () => {
+    const data = uuid ? await DayReportDAL.getDayReports(uuid) : [];
+    setDayReports(data);
+  };
+
+  useEffect(() => {
+    loadDayReports();
+  }, []);
+
+  /**
+   * Create day report
+   */
+  const createDayReport = async(id: string) => {
```

Comment:
stange function: it is not pure (loadDayReports), but also it required id as parameter. Choose the only one way. I recomment this signature: ```const createDayReport = async (wayId: string, onCreateWay: () => void): void => {...}

(and you will be able to move out it )

Reply:
I dont understand why need to use function onCreateWay inside Dayreports table

Reply:
Or you was mean function onCreateDayReport?

Reply:
I wrote signature, just implement it

Reply:
Done